docs/source/getting_started/v1_user_guide.md

```diff
@@ -0,0 +1,53 @@
+# vLLM V1 User Guide
+
+## Why vLLM V1?
+Previous blog post [vLLM V1: A Major Upgrade to vLLM's Core Architecture](https://blog.vllm.ai/2025/01/27/V1-alpha-release.html)
+
+## Semantic changes and deprecated features
+
+### Logprobs
+- vLLM V1 now supports both sample logprobs and prompt logprobs, as introduced in this [PR](https://github.com/vllm-project/vllm/pull/9880).
+- **Current Limitations**: 
+  - V1 prompt logprobs do not support prefix caching.
+  - V1 logprobs are computed before logits post-processing, so penalty 
+  adjustments and temperature scaling are not applied.
+- The team is actively working on implementing logprobs that include post-sampling adjustments.
+
+### The following features has been deprecated in V1:
+
+#### Deprecated sampling features
+- best_of
+- logits_processors
```

Comment:
- "Per request logits processors are not supported"
- Nick is working on supporting these globally in the server

Reply:
does that means both best_of and logits_processors are both not deprecated but will be supported globally?